R/g.getmeta.R

```diff
@@ -607,6 +607,41 @@ g.getmeta = function(datafile, params_metrics = c(), params_rawdata = c(),
           }
           CWav[h,1] = max(c(CW[h,1],CW[h,2],CW[h,3])) #indicator of clipping
         }
+        # Revise edges of nonwear score
+        # they might be imputed in g.imputeTimegaps and not revised for nonwear
+        # browser()
+        look4NWedges = NWav > 0
+        NWedges = diff(look4NWedges)
+        NWstarts = which(NWedges > 0) + 1
+        NWends = which(NWedges < 0)
+        windows2check = round(window/window2)
+        # check startpoint of nonwears
+        for (start_i in NWstarts) {
```

Comment:
Maybe add condition to check whether NWstarts has length larger than 0. This would apply for recordings with no non-wear but with timegaps that are too short to trigger the non-wear detection.